playbooks/common/openshift-master/config.yml

```diff
@@ -154,6 +154,7 @@
   - role: openshift_clock
   - role: openshift_cloud_provider
   - role: openshift_builddefaults
+  - role: openshift_buildoverrides
```

Comment:
This was a very recent change, rebase and move this to roles/openshift_master/meta/main.yml after openshift_builddefaults. Not sure why github isn't flaging this for needing a rebase.